app/migrations/MigrationRoutine.scala

```diff
@@ -53,7 +54,7 @@ class MigrationRoutine @Inject() (appConfig: AppConfig)(implicit mec: RoutinesEx
   private def migrateWithExportsMigrationTool(): Unit = {
     val lockManagerConfig = LockManagerConfig(lockMaxTries, lockMaxWaitMillis, lockAcquiredForMillis)
     val migrationsRegistry = MigrationsRegistry()
-      .register(new MakeParsedDetailsOptional())
+      .register(new PurgeExpiredNotifications())
```

Comment:
The convention we follow is to not replace the call to register but just add your new one below in a separate call to `register`